Clarified info about domain delegation (#3814)
magento/devdocs: guides/v2.1/cloud/cdn/configure-fastly.md

Fastly only allows you to assign an apex domain and associated subdomains to one Fastly service and account. If you have an existing Fastly account that links the same apex and subdomains used for your {{ site.data.var.ece }}, remove those domains from the existing account before submitting the request to add Fastly services to your project environments. See [Working with Domains](https://docs.fastly.com/guides/basic-configuration/working-with-domains) in the Fastly documentation.

If you have other sites hosted on the same apex subdomain and want to continue managing Fastly services from the existing account, you can leave the apex domain linked to that account and transfer the subdomains for your project to the {{ site.data.var.ece }} Fastly service account. To use this type of configuration, you must submit a Fastly support ticket requesting delegation of the apex domain.

For example, for a project with the following conditions and requirements, you would submit a [Fastly support ticket](https://fastly.zendesk.com/hc/en-us/articles/204950779-Filing-support-tickets) to delegate `testweb.com` to Magento, including the {{ site.data.var.ece }} project ID in the ticket:

> -  The {{ site.data.var.ece }} project uses the `storeprod.testweb.com` and `storestaging.testweb.com` subdomains

Reviewer: These bullet items don't use a consistent pattern

Author: Mainly for review-- total revision based on technical review

### Multiple Fastly accounts and assigned domains {#domain}

Fastly only allows you to assign an apex domain and associated subdomains to one Fastly service and account. If you have an existing Fastly account that links the same apex and subdomains used for your {{ site.data.var.ece }}, you have the following options:

-  Remove the apex and subdomains from the existing account before submitting the request to add Fastly services to your project environments. See [Working with Domains](https://docs.fastly.com/guides/basic-configuration/working-with-domains) in the Fastly documentation.

   Use this option to link the apex domain and all subdomains to the Fastly service account for {{ site.data.var.ece }}. 

> - Submit a support ticket to request domain delegation, designating which account manages the apex and each subdomain.

Author: ```suggestion
- Submit a support ticket to request domain delegation so that apex and subdomains can be linked to different accounts. 
```

  Use this option if your apex domain has multiple subdomains for Magento and non-Magento sites that you want to link to different Fastly accounts.

#### To request domain delegation

**Scenario 1**

If an apex domain (for example, `testweb.com` and `www.testweb.com`) is linked to an existing Fastly account, and you need to link your project subdomains to the Fastly account for {{ site.data.var.ece }}, submit a Fastly support ticket requesting that these subdomains (for example, `mcstaging.testweb.com` and `mcprod.testweb.com`) be delegated from the existing account to the Magento account. Include your Magento project ID in the ticket.  

**Scenario 2**

> If your apex domain (for example,`testweb.com` and `www.testweb.com`) links to a {{ site.data.var.ece }} Fastly service account, and you need to link associated subdomains (for example,`service.testweb.com` and `news.testweb.com`) to another Fastly account, submit a [Magento support ticket](https://support.magento.com/hc/en-us/articles/360000913794#submit-ticket) requesting that these subdomains be delegated from the {{ site.data.var.ece }} Fastly service account to the Fastly account. Include your the service ID for the Fastly account in the ticket.

Author: ```suggestion
If your apex domain (for example,`testweb.com` and `www.testweb.com`) links to a {{ site.data.var.ece }} Fastly service account, and you need to link associated subdomains (for example,`service.testweb.com` and `news.testweb.com`) to another Fastly account, submit a [Magento support ticket](https://support.magento.com/hc/en-us/articles/360000913794#submit-ticket) requesting that these subdomains be delegated from the {{ site.data.var.ece }} Fastly service account to the Fastly account. Include the service ID for the Fastly account in the ticket.
```

### Multiple Fastly accounts and assigned domains {#domain}

Fastly only allows you to assign an apex domain and associated subdomains to one Fastly service and account. If you have an existing Fastly account that links the same apex and subdomains used for your {{ site.data.var.ece }}, you have the following options:

> -  Remove the apex and subdomains from the existing account before submitting the request to add Fastly services to your project environments. See [Working with Domains](https://docs.fastly.com/guides/basic-configuration/working-with-domains) in the Fastly documentation.

Author: ```suggestion
-  Remove the apex and subdomains from the existing account before requesting Fastly service credentials for your {{ site.data.var.ece}} project environments. See [Working with Domains](https://docs.fastly.com/guides/basic-configuration/working-with-domains) in the Fastly documentation.
```
